src/components/15Years/15YearsStyles.vue

```diff
@@ -31,23 +31,32 @@
 		@include body-text();
 	}
 
```

Comment:
Since the CSS in this file isn't scoped these changes are effecting sections lower on the page. @rally-adam Can you please move these changes into 15YearsHeader.vue

Reply:
@ryan-ludwig Have you done this yet? I was going to fix some stylelint warnings from this file

Reply:
@ryan-ludwig done